### DISCUSSION TITLE
URL encoding

### DISCUSSION THREAD

**Discussion by ghost** [basdijkstra/rest-assured-net#123] [Q&A]
Hello, is there any possibility to enable url encoding as it was in Java?

**Comment by basdijkstra** [comment]
Hey @Alexander-Tribushny-DataArt , can you give me an example or some more documentation, so I know exactly what it is you're looking for?

Happy to see what I can do but I need a little more information here :)

**Comment by basdijkstra** [comment]
@Alexander-Tribushny-DataArt any further information or an example? I’d be happy to look into this but I’ll need some more details first.
